.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+## Jira Ticket:
+
+## Description of Work
+
+## Acceptance Criteria
+
+-
+
+## How to Test
+
+-
+
+### List of Changes:
```

Comment:
Is there a reason this is h3 compared to the others

Reply:
Just since the list of changes are less important since the description and changelog from GitHub will cover this

Reply:
Seems like good reasoning to me